risedev.yml

```diff
@@ -652,6 +671,9 @@ template:
     # Minio instances used by this compute node
     provide-minio: "minio*"
 
+    # AWS s3 bucket used by this compute node
```

Comment:
Should we provide correct documentation?

Reply:
Sorry I missed this typo, will fix it in next PR.